src/Features/Core/Portable/CodeRefactorings/SyncNamespace/AbstractSyncNamespaceCodeRefactoringProvider.cs

```diff
@@ -22,6 +22,12 @@ public override async Task ComputeRefactoringsAsync(CodeRefactoringContext conte
             var textSpan = context.Span;
             var cancellationToken = context.CancellationToken;
 
+            var workspace = document.Project.Solution.Workspace;
```

Comment:
the variable doesn't seem necessary.

Reply:
fixed in https://github.com/dotnet/roslyn/pull/35225/commits/ae12b5e88c4bf487f32f380024dd66db54664794